### DISCUSSION TITLE
Throttling issue

### DISCUSSION THREAD

**Discussion by casuffitsharp** [vcsjones/AzureSignTool#264] [General]
We're experiencing throttling issues with Azure Key Vault while using the latest release (5.0.0). It appears that the retry policy introduced in #251 might resolve this problem. Could you please let us know if there is a new official release planned that includes this fix?

**Comment by flobadob** [comment]
We're having the same issue. We send 200+ assemblies in one go. 

trce: AzureSignTool.SignCommand[0]
      Retrieving certificate sc-uks-dev-codesigning.
trce: AzureSignTool.SignCommand[0]
      Retrieved certificate sc-uks-dev-codesigning.
trce: AzureSignTool.SignCommand[0]
      Creating context
info: AzureSignTool.SignCommand[0]
      => File: PGL\Output\Assemblies\PGL.IP.Resources.dll
      Signing file.
trce: AzureSignTool.SignCommand[0]
      => File: PGL\Output\Assemblies\PGL.IP.Resources.dll
      Getting SIP Data
trce: AzureSignTool.SignCommand[0]
      => File: PGL\Output\Assemblies\PGL.IP.Resources.dll
      Calling SignerSignEx3
fail: AzureSignTool.SignCommand[0]
      => File: PGL\Output\Assemblies\PGL.IP.Resources.dll
      Signing failed with error 80070005.
info: AzureSignTool.SignCommand[0]
      => File: PGL\Output\Assemblies\PGL.IP.Resources.dll
      Stopping file signing.
info: AzureSignTool.SignCommand[0]
      Successful operations: 0
info: AzureSignTool.SignCommand[0]
      Failed operations: 1
